mbed_host_tests/host_tests_plugins/module_reset_mps2.py

```diff
@@ -67,12 +69,24 @@ def execute(self, capability, *args, **kwargs):
         @return Capability call return value
         """
         result = False
+
+        if not kwargs['destination_disk']:
+            self.print_plugin_error("Error: destination_disk not specified")
+            return False
+
+        destination_disk = kwargs.get('destination_disk', None)
+
+        # This optional parameter can be used if TargetID is provided (-t switch)
+        target_id = kwargs.get('target_id', None)
+        pooling_timeout = kwargs.get('polling_timeout', 60)
         if self.check_parameters(capability, *args, **kwargs) is True:
 
             if capability == 'reboot.txt':
-                # TODO: Implement touch file for reboot
-                pass
-
+                reboot_file_path = os.path.join(destination_disk,capability)
```

Comment:
Nit pick for the next few lines here, could you please add a space between function parameters and commas?